tests/test_lint.py

```diff
@@ -336,3 +336,27 @@ def test_bug_number_pings(content, num_nits):
     assert len(nits) == num_nits
     if num_nits > 0:
         assert set(["BUG_NUMBER"]) == set(v.check_name for v in nits)
+
+
+def test_redundant_pings():
+    """
+    Test that name contains '-ping' or 'ping-' or 'ping' or 'custom' yields lint errors.
+    """
+    content = [
+        {
+            "ping_test": {
+                "description": "testing the ping in the prefix",
+                "send_if_empty": True,
+                "bugs": ["https://bugzilla.mozilla.org/show_bug.cgi?id=1703577"],
+                "data_review": ["https://bugzilla.mozilla.org/show_bug.cgi?id=1703577"],
+            }
+        }
+    ]
```

Comment:
This right now fails because `content` needs to be an object, not a list.
Additionally it turns out `add_required_ping` adds most of these fields for us.
So this becomes:

```suggestion
    content = {
        "ping": {}
    }
```

If you really want to expand this test you can parametrize that test just as the one above, using `@pytest.mark.parametrize`.
This allows you to specify a list of inputs this test function is run with.

This is fully optional though, it's also ok if you leave the test barebones for now.

Reply:
I see, I changed `content` type from list to object and added an barebone `ping` object